README.md

```diff
@@ -142,6 +142,10 @@ wrapper of `ikfast.cpp` to the kinematics base interface of moveit.
 Contains a specialized version of `ros_controllers::JointTrajectoryController` which can be put into a holding mode.
 A controlled stop using a hold trajectory is performed thus stopping the manipulator without the mechanical stress of a hard brake.
 
+**Please note:**
+Due to safety reasons, the command interface of the `PilzJointTrajectoryController` is deactivated. For more information
+see issue [#493](https://github.com/ros-controls/ros_controllers/issues/493) on ros-controls/ros_controllers.
```

Comment:
```suggestion
The topic interface of the original `joint_trajectory_controller` is deactivated. Please use the action interface to send goals, that allows monitoring: The sender gets notified about cancelled goals.
```
Instead of highlighting the reason, focussing on the benefits of the action interface and suggesting that as a solution. Maybe link to http://wiki.ros.org/joint_trajectory_controller#Available_interfaces

Reply:
https://github.com/PilzDE/pilz_robots/pull/405/commits/e764f9391691e157b7e17faec02e403037480a2e